<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Board</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>5</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f0f0f0; border: 1px solid #ccc;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <layout class="QVBoxLayout" name="teamLayout">
        <item>
         <widget class="QLabel" name="labelTeam1">
          <property name="text">
           <string>Team Red</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_team1"/>
        </item>
        <item>
         <widget class="QLabel" name="labelTeam2">
          <property name="text">
           <string>Team Blue</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_team2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="modeLayout">
        <item>
         <widget class="QLabel" name="labelMode">
          <property name="text">
           <string>Game Mode</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Mode">
          <item>
           <property name="text">
            <string>Human vs Human</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Human vs AI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auto Play</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Start">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Reset">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Save">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Replay">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Replay</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
